src/main/java/DateAndTime.java

```diff
@@ -0,0 +1,34 @@
+public class DateAndTime {
+    protected int date;
+    protected int month;
+    protected int year;
+    protected int timeNumber;
+    protected String timeDay;
+
+    protected String[] monthMappings = new String[]{"", "January", "February", "March", "April", "May", "June",
+                                                    "July", "August", "September", "October", "November", "December"};
+
+    public DateAndTime(String dateAndTime) {
```

Comment:
Documentation on parse-able dateAndTime strings will help improve code readability

Reply:
I now have included Javadoc comments in later commits!